sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -392,6 +393,11 @@ public abstract partial class ClientOptions
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string? ToString() { throw null; }
     }
+    public partial class ClientRedirectsOptions
+    {
+        public ClientRedirectsOptions() { }
```

Comment:
I believe options types used on ClientOptions typically have an internal constructor, since customers don't need to create them.